ci/do_ci.sh

```diff
@@ -56,14 +56,11 @@ elif [[ "$1" == "cmake.exporter.prometheus.test" ]]; then
 #  apt-get install sudo
 #  apt-get install zlib1g-dev
 #  apt-get -y install libcurl4-openssl-dev
-  cd third_party
-  git clone https://github.com/jupp0r/prometheus-cpp
-  cd prometheus-cpp
-  git checkout v0.9.0
-  git submodule init
-  git submodule update
+  cd third_party/prometheus-cpp
+  git submodule update --recursive --init
```

Comment:
Right now we are tracking to latest release of `prometheus-cpp` as a submodule, and all tests pass. Recursive update needed because at top-level config we do shallow clonining, not recursive. We can change this later to deep recursive cloning.

Reply:
We should be fine now even without it - because I changed the YAML file for the GitHub Action to do deep recursive clone. But even with this - we are fine, since we'd just run thru the update without updating anything. It'll be the same as what's already fetched, exiting quite quickly from this line.